src/main/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSparkEngine.java

```diff
@@ -48,10 +48,35 @@ public BwaSparkEngine(final JavaSparkContext ctx,
 
     public SAMFileHeader getHeader() { return broadcastHeader.getValue(); }
 
-    public JavaRDD<GATKRead> align(final JavaRDD<GATKRead> unalignedReads) {
+    /**
+     * Performs pair-end alignment on a RDD.
+     * @param unalignedReads the read-pairs to align.
+     * @return never {@code null}.
+     */
+    public JavaRDD<GATKRead> alignPairs(final JavaRDD<GATKRead> unalignedReads) {
+        return align(unalignedReads, true);
+    }
+
+    /**
+     * Performs single-end alignment on a RDD.
+     *
+     * @param unalignedReads the reads to align.
+     * @return never {@code null}.
+     */
+    public JavaRDD<GATKRead> alignSingletons(final JavaRDD<GATKRead> unalignedReads) {
```

Comment:
Singleton doesn't seem to me to be quite the right word.  alignReads?  alignUnpairedReads?